warp-element/package.json

```diff
@@ -6,15 +6,27 @@
   "main": "./src/element.js",
   "files": [
     "./src",
-    "index.d.ts",
+    "types",
     "README.md"
   ],
   "types": "./index.d.ts",
```

Comment:
Might want to remove this now, or have a compilation step that also produces this type definition.

Reply:
@wkillerud Absolutely! Removed now 💣